docs/admin/how-to/setup-https.mdx

```diff
@@ -1,106 +1,226 @@
-# How to setup HTTPS connection with Ingress controller on your Kubernetes instance
+# How to setup HTTPS/TLS for your Kubernetes Sourcegraph instance
```

Comment:
I considered removing this entire file altogether but took a hack at a procedure instead -- this is generated as a best effort but I think maybe we ought to just not provide guidance here. @loujar what do you think?